egs/wsj/s5/steps/online/nnet2/copy_data_dir.sh

```diff
@@ -76,7 +76,7 @@ else
 fi
 
 
-for f in feats.scp segments wav.scp reco2file_and_channel text stm glm ctm; do
+for f in feats.scp segments wav.scp reco2file_and_channel text stm glm ctm offsets.scp offset_type; do
```

Comment:
is offset_type some kind of old code that shouldn't be there?  I don't see it elsewhere.